### PR TITLE
fix: tables not find in parentheses join sql.

### PR DESCRIPTION
**Pull request by Hanchers**
to fix bug #1955

### CODE REVIEW

**Comment by manticore-projects** [comment]
Thank you for your contribution and the fix, please see my comments.

**Comment by Hanchers** [comment]
> Thank you for your contribution and the fix, please see my comments. The Spotless Formatter also raised concerns.

fix and please review again

**Comment by manticore-projects** [comment]
> > Thank you for your contribution and the fix, please see my comments. The Spotless Formatter also raised concerns.
> 
> fix and please review again

I still see the "star" imports, please import Classes explicitly instead.

**Comment by Hanchers** [comment]
> > > Thank you for your contribution and the fix, please see my comments. The Spotless Formatter also raised concerns.
> > 
> > 
> > fix and please review again
> 
> I still see the "star" imports, please import Classes explicitly instead.

sorry, my idea have a auto import config that couse the "star" imports.  
this time i fix this question and please review again. 
thanks

File: src/test/java/net/sf/jsqlparser/util/TablesNamesFinderTest.java [hunk 1]
```diff
@@ -36,6 +25,16 @@
...
 
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.StringReader;
+import java.util.List;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (manticore-projects) [inline src/test/java/net/sf/jsqlparser/util/TablesNamesFinderTest.java:36]: Please fix this.

**Review by manticore-projects — CHANGES_REQUESTED** [review]

**Review by manticore-projects — APPROVED** [review]

**Comment by manticore-projects** [comment]
Thank you for the corrections and your contribution. I am going to merge it.

**Comment by Hanchers** [comment]
> Thank you for the corrections and your contribution. I am going to merge it.

thanks， my pleasure

**Pull request merged**